packages/trader/src/Stores/Modules/MT5/mt5-store.js

```diff
@@ -202,9 +206,20 @@ export default class MT5Store extends BaseStore {
     @action.bound
     setError(state, obj) {
         this.has_mt5_error = state;
+        this.error_type = obj?.code ?? '';
+
+        if (this.error_type === 'PasswordError') {
+            this.incrementFailedPasswordAttempts();
+        }
+
         this.error_message = obj ? obj.message : '';
     }
 
+    @action.bound
+    incrementFailedPasswordAttempts() {
+        this.failed_password_attempts += 1;
```

Comment:
where do we reset it back to 0?

Reply:
Turns out this was not needed at all, since server sends 2 different error codes. one is `PasswordError`, and the other one is `PasswordReset`. So we can entirely rely on the API for this. I've removed the variables.